casanovo/casanovo.py

```diff
@@ -382,7 +395,76 @@ def setup_model(
     return config, model
 
 
-def _get_model_weights() -> str:
+def _get_weights_from_url(
+    file_url: Optional[str],
```

Comment:
**question:** I don't see a default value for `file_url`, so why does it have an `Optional` type hint?

Reply:
My bad, I thought that the Optional type hint was similar to std::optional from C++, meaning in this case that the file_url may or may not be present.